protostar/commands/test/test_cases.py

```diff
@@ -17,13 +19,19 @@ class TestCaseResult:
     def get_formatted_file_path(self):
         return log_color_provider.colorize("GRAY", str(self.file_path))
 
+    @abstractmethod
+    def log(self, _: bool = False) -> str:
+        ...
+
 
 @dataclass(frozen=True)
 class PassedTestCase(TestCaseResult):
     test_case_name: str
     execution_resources: Optional[ExecutionResourcesSummary]
+    captured_setup_stdout: str
+    captured_test_stdout: str
 
-    def __str__(self) -> str:
+    def log(self, include_stdout_section: bool = False) -> str:
```

Comment:
maybe?

```suggestion
    def display(self, include_stdout_section: bool = False) -> str:
```

by the name `log` I thought I can send custom logs from testing code with this function

Reply:
Problem is we use `log` a lot as a name for such functions, I am worried about consistency. Agreed `display` is a better name though.

Reply:
Oh i didn't see nvm then